app/controllers/admin/links_controller.rb

```diff
@@ -28,11 +29,13 @@ def edit
 
     def update
       @link.update(link_params)
+      link_scope.invalidate_cache
       respond_with(@link, location: admin_links_path)
     end
 
     def destroy
       @link.destroy
+      link_scope.invalidate_cache
```

Comment:
@yakloinsteak I _think_ this handles the invalidation, look good to you?

Reply:
That looks great. I'm glad we could do it without redis in this particular case.